docs/scripting/directives/d.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "#D"
+---
+
+Debugging output function.
+
+#D - or 'Debug' - is a function that will take any content you input to it and add it to a debug log. At the end of the script run - if you are the script owner - all of the debug logs will be returned instead of the regular output. This output will be returned even if the script hits an uncatched error or ((%D:::TRUST COMMUNICATION:::%))
```

Comment:
```suggestion
#D - or 'Debug' - is a function that will take any content you input to it and add it to a debug log. At the end of the script run - if you are the script owner - all of the debug logs will be returned instead of the regular output. This output will be returned even if the script hits an uncaught error or ((%D:::TRUST COMMUNICATION:::%))
```